Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -82,26 +82,28 @@ else
   fi
 
 fi
+echo "BUCKET_UPLOAD = $BUCKET_UPLOAD, FORCE_BUCKET_UPLOAD = $FORCE_BUCKET_UPLOAD"
 
-if [ -z "${BUCKET_UPLOAD}" ]; then
+if [ -z "${BUCKET_UPLOAD}" ] && [ -z "${FORCE_BUCKET_UPLOAD}" ]; then
   echo "Updating the following content packs: $CONTENT_PACKS_TO_UPLOAD ..."
   python3 ./Tests/Marketplace/upload_packs.py -pa $PACK_ARTIFACTS -d $ARTIFACTS_FOLDER/packs_dependencies.json -e $EXTRACT_FOLDER -b $GCS_BUILD_BUCKET -s "$GCS_MARKET_KEY" -n "$CI_PIPELINE_ID" -p $CONTENT_PACKS_TO_UPLOAD -o false -sb $BUILD_BUCKET_PACKS_DIR_PATH -k $PACK_SIGNING_KEY -rt true -bu false -c $CI_COMMIT_BRANCH -f false -dz "$CREATE_DEPENDENCIES_ZIP" -mp "$MARKETPLACE_TYPE"
   echo "Finished updating content packs successfully."
 else
   # In Upload-Flow, we exclude test-pbs in the zipped packs
   REMOVE_PBS=true
-  BUCKET_UPLOAD_FLOW=true
   GCS_PRIVATE_BUCKET="marketplace-dist-private"
-  if [ -n "${FORCE_PACK_UPLOAD}" ] && [ -n "${PACKS_TO_UPLOAD}" ]; then
+  if [ -n "${FORCE_BUCKET_UPLOAD}" ] && [ -n "${PACKS_TO_UPLOAD}" ]; then
     # In case the workflow is force upload, we override the forced packs
     echo "Force uploading to production the following packs: ${PACKS_TO_UPLOAD}"
     OVERRIDE_ALL_PACKS=true
     CONTENT_PACKS_TO_UPLOAD="${PACKS_TO_UPLOAD}"
     IS_FORCE_UPLOAD=true
+    BUCKET_UPLOAD_FLOW=false
```

Comment:
@moishce - This is to avoid commits comparations etc.? did you verify that it doesn't avoid other flows/logics to run during the upload to the build bucket?

Reply:
yes